libs/components/indicators/testing/src/tokens/token-harness.ts

```diff
@@ -9,7 +9,7 @@ import { SkyTokenHarnessFilters } from './token-harness-filters';
 export class SkyTokenHarness extends ComponentHarness {
   public static hostSelector = 'sky-token';
 
-  #getDismissButton = this.locatorForOptional('button.sky-token-btn-close');
+  #getDismissButton = this.locatorFor('button.sky-token-btn-close');
```

Comment:
This selector can sometimes return `null` if the token is not dismissible.

Reply:
True, but the only place where we call it is right after we check to see if the token is dismissible. I changed this code so I wouldn't have to wrap the click logic in an `if` statement that I couldn't reach with a unit test.